active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,320 @@
+# Number box
+
+## Background
+Xaml has a TextBox control for text input, which can be used for numeric input, but numeric input scenarios benefit from custom features, such as buttons to increment/decrement. The NumberBox control in this spec provides these features. This will ship as part of the [WinUI package](https://www.nuget.org/packages/Microsoft.UI.Xaml), not as part of the Windows OS.
```

Comment:
Nit: Can you line-wrap these at 80ish so it's easier to comment on pieces?

Reply:
The problem with that is that it defeats GitHub's Insert Suggestion feature, which lets you suggest a rewrite of one line only.